### PR TITLE
Hotfix/APPEALS-23420

### PR DESCRIPTION
**Pull request by AimanK**
Resolves [APPEALS-23420](https://jira.devops.va.gov/browse/APPEALS-23420)

# Description
Issue is timeout is reached prior to returned results. There is no reportable issue/error from logs. Only client side error as shown below. This is due to the high amount of code execution time in VACOLS which causes the 60 second timeout to expire and the client code throws an error stating a server error has occurred.

Suggested fixes:
1. Increase timeout - could effect user experience due to longer wait times to search for a veteran
2. Limit search results to 10 and page results with subsequent calls to VACOLS in a job after the first 10 have been fetched. - This would be the best route forward limiting the impact on user experience.

Front end errors: View chrome dev tools to see this error when searching for a veteran.

Current limited workaround for support is adding URLs to each incident ticket so the users can access
https://query.prod.appeals.va.gov/dashboard/206-veteran-appeal-urls

```
{
    "errorType": "UNKNOWN_SERVER_ERROR"
} 

```
## Acceptance Criteria
- [x] Code compiles correctly

## Testing Plan
1. Go to [TBD](https://jira.devops.va.gov/browse/APPEALS-23420) or list them below

- [x] For feature branches merging into master: Was this deployed to UAT?

### CODE REVIEW

**Review by raymond-hughes — APPROVED** [review]

File: db/schema.rb [hunk 1]
```diff
@@ -1652,6 +1652,7 @@
     t.boolean "national_cemetery_administration", default: false
     t.boolean "no_special_issues", default: false, comment: "Affirmative no special issues, added belatedly"
     t.boolean "nonrating_issue", default: false
+    t.boolean "pact_act", default: false, comment: "The Sergeant First Class (SFC) Heath Robinson Honoring our Promise to Address Comprehensive Toxics (PACT) Act"
```
Comment (ThorntonMatthew) [inline db/schema.rb:1655]: These schema changes look like they may be stowaways from a previous migration/reversion. The MST/PACT work has not been released yet, and as a result I do not believe that this should be included.

**Review by ThorntonMatthew — CHANGES_REQUESTED** [review]
I've left a comment regarding the schema diff.

Also, have the failures in `spec/feature/queue/case_details_spec.rb` been looked into? I do not see the ones related to missing table elements listed in the [flaky tests epic](https://jira.devops.va.gov/browse/APPEALS-24070) so it is unclear if an investigation has taken place.

If these items are deemed to not be of concern, then please let me know and I'll flip my review. 😄 We will likely be deploying work on Thursday alongside this code which is why I was poking around a bit.

**Review by raymond-hughes — APPROVED** [review]

File: db/schema.rb (hunk 1 above)
Comment (raymond-hughes) [reply db/schema.rb]: @ThorntonMatthew We are reverting the schema file back.

**Review by ThorntonMatthew — APPROVED** [review]
Thank you for being willing to make some tweaks to this branch. My concerns are abated.

**Pull request merged**